cpp/src/strings/find_multiple.cu

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cudf/column/column_factories.hpp>
+#include <cudf/column/column_device_view.cuh>
+#include <cudf/strings/find_multiple.hpp>
+#include <cudf/strings/strings_column_view.hpp>
+#include <cudf/strings/string_view.cuh>
+#include <cudf/utilities/error.hpp>
+#include <strings/utilities.hpp>
+
+#include <thrust/transform.h>
+
+namespace cudf
+{
+namespace strings
+{
+namespace detail
+{
+
+std::unique_ptr<column> find_multiple( strings_column_view const& strings,
+                                       strings_column_view const& targets,
+                                       rmm::mr::device_memory_resource* mr = rmm::mr::get_default_resource(),
+                                       cudaStream_t stream=0 )
+{
+    auto strings_count = strings.size();
+    if( strings_count==0 )
+        return make_empty_strings_column(mr,stream);
+    auto targets_count = targets.size();
+    if( targets_count==0 )
+        return std::make_unique<column>(strings.parent());
```

Comment:
Why parent()? What does this do?

Reply:
Argh, this is a mistake. I need to create a test case for this.